src/js/components/Products/Product.js

```diff
@@ -0,0 +1,187 @@
+                <Navbar titleFromParent={loc_navBarTitle} navbarItems={loc_navbarItems}/>
+                <div className="container ">
+                    <div className="row marginTop20px ">
+                            <h3 className="marginLeft20px"></h3>
+                    </div>
+                    <div>
+                        <img src={this.state.url || "http://placehold.it/150x150" } alt="Uploaded images" height="150" width="150"></img>
+                    </div>
+                    <div className="row ">
+                        <div className="col-12 marginTop20px">
+                            <label >Cover Image</label>
+                            <span>
+                                <input name="file" type="file"  onChange={this.handleChage}/> 
+                             </span>
+                        </div>
+                        <div className="col-12 marginTop20px">
+                            <label >Name</label>
+                            <span>
+                                <input className="formInput" name="Name" type="text" onChange={this.onChange}  value={this.state.Name}/>
+                            </span>
+                        </div>
+                        <div className="col-12 marginTop20px">
+                            <label >Description</label>
+                        </div>
+                        <div className="col-12 marginTop10px">
+                            <textarea className ="Textarea"  type="text" name ="Description"  onChange={this.onChange}  value={this.state.Description}/>
+                        </div>
+                        <div className="col-12 marginTop20px">
+                            <label >Type/Category</label>
+                        </div>
+                        <div className="col-12 marginTop10px">
+                            <span className ="rad">
+                                <input type="radio" name="Speedy" value={this.state.Speedy} onClick={this.handleChecked}  /><label >Speedy</label>
+                                <input type="radio" value="Know your nation" name="Speedy" value={this.state.Speedy} onClick={this.handleChecked}/> Know your nation
+                            </span>
+                        </div>
+                        <div className="col-12 marginTop20px">
+                            <label >Price</label>
+                            <span>
+                                <input className="formInput" name="Price" type="Price" onChange={this.onChange}  value={this.state.Price}/>
+                            </span>
+                        </div>
+                        <div className="col-12 marginTop20px">
+                            <label >Variations</label>
+                        </div>
+                        <div className="col-12 marginTop10px">
+                            <label >Size</label>
+                        </div>
+                        <div className="col-12 marginTop10px">
```

Comment:
can you not do inline styling? unless it's bootstrap classes
~~marginTop10px~~

Reply:
yeah its bootstrap classes